zeebe/broker-client/src/test/java/io/camunda/zeebe/broker/client/api/BrokerClientTest.java

```diff
@@ -65,6 +66,7 @@ public final class BrokerClientTest {
 
   @BeforeEach
   void beforeEach() {
+    final var meterRegistry = new SimpleMeterRegistry();
```

Comment:
:wrench:  should be closed in `@AfterEach`

Reply:
Do you know if it actually leaks resources, or just puts pressure on the GC?